### PR TITLE
Adapter master data helm chart

### PR DESCRIPTION
**Pull request by pintu-eGov**

### CODE REVIEW

File: deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# ingress.yaml
+{{- template "common.ingress" . -}}
```
Comment (rushang7) [inline deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/templates/ingress.yaml]: We need ingress only when there are api calls coming from outside the kube cluster.

File: deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Common Labels
+labels:
+  app: "adapter-master-data"
+  group: "rainmaker"
+
+# Ingress Configs
+ingress:
+  enabled: true
+  zuul: false
```
Comment (rushang7) [inline deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml]: When zuul is marked false, the api call from outside will directly reach the server. So standard protections like, Authn&AuthZ, Rate Limiting, etc. will get skipped, due to which there are chances of an attack.

File: deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  dbMigration:
+    enabled: false
+    
+# Container Configs
+image:
+  repository: "adapter-master-data"
+replicas: "1"
+healthChecks:
+  enabled: false
+  livenessProbePath: "/adapter-master-data/health"
```
Comment (rushang7) [inline deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml]: Are these endpoints present in the service?

File: deploy-as-code/helm/environments/mgramseva-qa-secrets.yaml [hunk 4]
```diff
@@ -5,6 +5,9 @@ cluster-configs:
             password: ENC[AES256_GCM,data:vPC6D29TkTcwtw66g/FSaw==,iv:CcTl/3Ez01EbaiQZitlb4hwhVmqmjP8nNH0jWnF163k=,tag:OV/czZSWCauZmWQ38Ip50A==,type:str]
             flywayUsername: ENC[AES256_GCM,data:i7BiRT7z,iv:oar3tezcuh0hLw7moc8eMxTW9v41dkb5AJxxybYd50o=,tag:BG9DbFV4UTxP+f4uG4yXhw==,type:str]
             flywayPassword: ENC[AES256_GCM,data:W2OxvhXREj4EYp+bl7wb/A==,iv:cSU57AgKr9kaW8/zw0FH7y+DCbn9FFkthy0WLLosp68=,tag:hoNO09iWmdrPwszc4iPdig==,type:str]
+            mongo-db-username: ENC[AES256_GCM,data:jCYkLFcbNZwy9A==,iv:QKp+PAh6snZ8gDJZ1KhdLb210PsL0N1dTdzJUCkbl0s=,tag:p4cKNgJuavzLNu6giQmtRA==,type:str]
```
Comment (rushang7) [inline deploy-as-code/helm/environments/mgramseva-qa-secrets.yaml]: The secrets can't be copied by simple copy-paste. They have signature associated with them(present at the end of the file). You need to request DevOps team to copy them.

File: deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  - name: IFIX_MASTER_EXPENDITURE_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-service-host
+        key: adapter-master-data-service
+  - name: IFIX_DEPARTMENT_ENTITY_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-service-host
+        key: adapter-department-entity-service
```
Comment (rushang7) [inline deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml]: We will let the name be ifix-department-entity-service.

File: deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml [hunk 6]
```diff
@@ -0,0 +1,64 @@
+# Common Labels
+labels:
+  app: "adapter-master-data"
+  group: "rainmaker"
+
```
Comment (rushang7) [inline deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml:5]: namespace key is missing.

**Review by rushang7 — CHANGES_REQUESTED** [review]

**Review by rushang7 — APPROVED** [review]

**Review by rushang7 — APPROVED** [review]

**Review by sathishp-eGov — APPROVED** [review]

**Pull request merged**